src/services/shims.ts

```diff
@@ -1060,8 +1060,15 @@ namespace ts {
             return this.forwardJSONCall(`resolveModuleName('${fileName}')`, () => {
                 const compilerOptions = <CompilerOptions>JSON.parse(compilerOptionsJson);
                 const result = resolveModuleName(moduleName, normalizeSlashes(fileName), compilerOptions, this.host);
+                const resolvedFileName = result.resolvedModule ? result.resolvedModule.resolvedFileName : undefined;
+                if (resolvedFileName && !compilerOptions.allowJs && fileExtensionIs(resolvedFileName, ".js")) {
+                    return {
+                        resolvedFileName: undefined,
+                        failedLookupLocations: undefined

```

Comment:
can you return empty array instead? I don't recall if VS side expects this property to always be set.